### PR TITLE
Paths

### PR DESCRIPTION
**Pull request by therfert**
##### SUMMARY
  - api_info, api_modify - support API paths - ``interface ethernet poe``, ``interface gre6``, ``interface vrrp`` and also support for all previously missing fields of entries in ``ip dhcp-server``

  - api_info, api_modify - ``address-pool`` field of entries in API path ``ip dhcp-server`` is not required anymore

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
api_*

### CODE REVIEW

File: changelogs/fragments/137-api.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - api_info, api_modify - support API paths - ``interface ethernet poe``, ``interface gre6``, ``interface vrrp`` and also support for all preivously missing fileds of entries in ``ip dhcp-server``
```
Comment (felixfontein) [inline changelogs/fragments/137-api.yml:3]:
```suggestion
  - api_info, api_modify - support API paths ``interface ethernet poe``, ``interface gre6``, ``interface vrrp`` and also support all previously missing fields of entries in ``ip dhcp-server`` (https://github.com/ansible-collections/community.routeros/pull/137).
```

File: changelogs/fragments/137-api.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+bugfixes:
+  - api_info, api_modify - ``address-pool`` field of entries in API path ``ip dhcp-server`` is not required anymore
```
Comment (felixfontein) [inline changelogs/fragments/137-api.yml:5]: The required should only affect api_modify, api_info shouldn't care about it:
```suggestion
  - api_modify - ``address-pool`` field of entries in API path ``ip dhcp-server`` is not required anymore (https://github.com/ansible-collections/community.routeros/pull/137).
```

File: changelogs/fragments/137-api.yml (hunk 1 above)
Comment (therfert) [reply changelogs/fragments/137-api.yml:3]: ohh right, I was tired yesterday:)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@therfert thanks for your contribution!

**Pull request merged**